### ISSUE TITLE
Non Inclusive Language Keywords

### ISSUE THREAD

**Issue by mcleo-d** [finos/InnerSource#49]
### Description

This is a notification email from LFX Security regarding the InnerSource Special Interest Group project.

During a recent code scan we discovered some of the repository files have Non Inclusive Language Keywords.

Please review the issues identified below. If possible, consider using alternative wording, variable names, and/or labels. If one or more of these issues are false positives, navigate to the LFX Security console -> InnerSource Special Interest Group -> Non-Inclusive Language and dismiss the alert.

[.github/ISSUE_TEMPLATE/meeting-minutes.md#L19](https://github.com/finos/innersource/blob/355382b4c455fb8200f6bea12f1d0d04833d0926/.github/ISSUE_TEMPLATE/meeting-minutes.md#L19)
Non-inclusive Language - master

**Comment by amaltson** [comment]
Right, I saw this too, it comes down to the our main branch still being called `master`. Our docs just talk about that main branch, so to solve the issue we need to rename the main branch. If you're OK with that, I can take that change on and rename it to the standard `main`.

**Comment by claredillon** [comment]
Sounds good @amaltson
